src/api/VoteMonitor.Api.Observer/Commands/RemoveDeviceIdCommand.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using AutoMapper;
+using VoteMonitor.Api.Observer.Models;
+
+namespace VoteMonitor.Api.Observer.Commands
+{
+    public class RemoveDeviceIdCommand : IRequest<bool>
```

Comment:
Suggesting to `IRequest` only as there is no additional value to have a flag for success/failure

Reply:
Could you please rephrase this one? Kinda new to .NET 😅

Reply:
sure, apologies; so `RemoveDeviceIdCommand ` implements `IRequest<bool>` . `bool` here is used to specify the return type of the Handle method. It returns `true` when we have a success, for example when the update was successful at the database level and `false` when there was some exception. But we don't really need to use it as in C# we also have try/catch/throw to control the exceptions flow, and I personally see it actually as an antipattern (imagine all methods returning a bool or an int or something else to denote the success or failure).

so what works very well here is to just use
`public class RemoveDeviceIdCommand : IRequest`

I noticed this code is similar to the `ResetDeviceCommand` one, but that is not really the most appropriate one to follow. I would suggest inspiring from `CreatePollingStationInfo` command.